docs/editors.json

```diff
@@ -1,18 +1,43 @@
 {
     "arcade": {
-        "name": "Arcade BETA",
+        "name": "Arcade",
```

Comment:
This will break the GitHub explorer which requires beta.

Reply:
I don't understand. There is BETA editor too in this list.

Reply:
The key is used also used as the target id when resolving an editor for a github repo. Once we ship the update, we can have "arcade" point to the released version.

Reply:
Ok fixed, I need to remember to fix this once we relase.